templates/services/merge/distributedValidator/charon.tmpl

```diff
@@ -0,0 +1,24 @@
+{{/* charon.tmpl */}}
+{{ define "distributedValidator" }}
+  charon:
+    image: obolnetwork/charon:${DV_IMAGE_VERSION}
+    environment:
+      - CHARON_BEACON_NODE_ENDPOINTS=${CC_API_URL}
+      - CHARON_LOG_LEVEL=${DV_LOG_LEVEL:-info}
+      - CHARON_LOG_FORMAT=${DV_LOG_FORMAT:-console}
+      - CHARON_P2P_RELAYS=${DV_P2P_RELAYS:-https://0.relay.obol.tech}
```

Comment:
Lets add a second relay here so if anything happens to 0, these aren't impacted. 

```suggestion
      - CHARON_P2P_RELAYS=${DV_P2P_RELAYS:-https://0.relay.obol.tech,https://1.relay.obol.tech}
```